build.gradle

```diff
@@ -311,22 +311,24 @@ configure(javaCodeCheckedProjects) {
 
     spotbugs {
         excludeFilter = new File(configDir, 'findbugs-exclude.xml')
-        sourceSets = [sourceSets.main]
-        toolVersion = '3.1.7'
     }
 
     codenarc {
-        toolVersion = '1.1'
+        toolVersion = '1.6.1'
         reportFormat = project.buildingWith('xmlReports.enabled') ? 'xml' : 'html'
     }
 
-    tasks.withType(com.github.spotbugs.SpotBugsTask) {
+    spotbugsMain {
         reports {
             xml.enabled = project.buildingWith('xmlReports.enabled')
             html.enabled = !project.buildingWith('xmlReports.enabled')
         }
     }
 
+    spotbugsTest {
```

Comment:
It complained about the use of sourceSets up above so using this as an alternative.

Reply:
Ack